docs/scripting/functions/PlayerGangZoneStopFlash.md

```diff
@@ -0,0 +1,64 @@
+---
+title: PlayerGangZoneStopFlash
+description: Stop player gangzone flash
+tags: ["player", "gangzone", "playergangzone"]
+---
+
+## Description
+
+Stop player gangzone flash.
+
+| Name        | Description                                                      |
+| ----------- | ---------------------------------------------------------------- |
+| playerid    | The ID of the player to whom player gangzone is bound.           |
+| zoneid      | The ID of the player gangzone for in stop flashing.              |
+
+## Returns
+
+1: The function executed successfully. Success is reported even if the player gangzone wasn't flashing to begin with.
+
+0: The function failed to execute. The gangzone specified does not exist.
+
+## Examples
+
+```c
+// This variable is used to store the id of the gangzone
+// so that we can use it throught the script
+new gGangZoneID[MAX_PLAYERS] = {INVALID_GANG_ZONE, ...};
+
+public OnPlayerConnect(playerid)
+{
+    // Create the gangzone
+    gGangZoneID[playerid] = CreatePlayerGangZone(playerid, 2236.1475, 2424.7266, 2319.1636, 2502.4348);
+
+    // Show the gangzone player
+    PlayerGangZoneShow(playerid, gGangZoneID[playerid]);
+
+    // Start player gangzone flash    
+    PlayerGangZoneFlash(playerid, gGangZoneID[playerid], 0xFF00FFFF);
+}
+
+public OnPlayerDeath(playerid, killerid, reason)
+{
+    // Check if gangzone exists and also check if it flashing
+    // If true, then stop flashing gangzone
+    if(gGangZoneID[playerid] != INVALID_GANG_ZONE && IsPlayerGangZoneFlashing(playerid, gGangZoneID[playerid]))
+        PlayerGangZoneStopFlash(playerid, gGangZoneID[playerid]);
```

Comment:
```suggestion
```

Reply:
I think it’s worth using `IsValidPlayerGangZone `here too instead of `gGangZoneID[playerid] != INVALID_GANG_ZONE`.

Reply:
Sorry, I’m still not very familiar with this Github interface.